Domain/src/main/java/starterProject/App.java

```diff
@@ -38,5 +39,7 @@ public static void main(String[] args) {
 
     }
 
+   public static List<Car> lists = createCar();
```

Comment:
Not needed

Reply:
Still not needed, remove

Reply:
Still not needed, remove please. You shouldn't be referencing the App class in any other class. App is just the entry point for your application and should interact with classes.

Reply:
Still here....